ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -145,7 +157,9 @@ public class DrivetrainSubsystem extends SubsystemBase {
   /**
    * Constructor for this DrivetrainSubsystem
    */
-  public DrivetrainSubsystem() {
+  public DrivetrainSubsystem(SubsystemCollection subsystems) {
+    cameraSubsystem = subsystems.getCameraSubsystem();
```

Comment:
Need to couch this in an if block properly.  Subsystems come and go around here.  We need to make sure it is present before assignment.

Reply:
added a installed hardware check